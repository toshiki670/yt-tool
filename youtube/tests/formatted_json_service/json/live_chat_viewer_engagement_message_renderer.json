{
  "replayChatItemAction": {
    "actions": [
      {
        "addChatItemAction": {
          "item": {
            "liveChatViewerEngagementMessageRenderer": {
              "id": "ChwKGkNJQ2RucnJCaUlzREZkM0N3Z1FkcUJRVTJR",
              "icon": { "iconType": "POLL" },
              "message": {
                "runs": [
                  { "text": "新しいゲームはどうですか？", "bold": true },
                  { "text": "\n" },
                  { "text": "持ってる (41%)" },
                  { "text": "\n" },
                  { "text": "持ってない (23%)" },
                  { "text": "\n" },
                  { "text": "買うか検討中 (17%)" },
                  { "text": "\n" },
                  { "text": "絶対に買わない (17%)" },
                  { "text": "\n" },
                  { "text": "\n" },
                  { "text": "Poll complete: 34 votes" }
                ]
              },
              "contextMenuEndpoint": {
                "commandMetadata": {
                  "webCommandMetadata": { "ignoreNavigation": true }
                },
                "liveChatItemContextMenuEndpoint": {
                  "params": "Q2g0S0hBb2FRMGxEWkc1eWNrSnBTWE5FUm1RelEzZG5VV1J4UWxGVk1sRWFLU29uQ2hoVlF6WnZURGhuWlV0TlpYbG5iUzAwT1dsTWNuRmxabmNTQ3pOcU5pMHpTakpXZUdKVklBRW9CRElhQ2hoVlF6WnZURGhuWlV0TlpYbG5iUzAwT1dsTWNuRmxabmM0QWtnQVVCWSUzRA=="
                }
              },
              "contextMenuAccessibility": {
                "accessibilityData": { "label": "Chat actions" }
              }
            }
          },
          "clientId": "CICdnrrBiIsDFd3CwgQdqBQU2Q"
        }
      }
    ]
  },
  "videoOffsetTimeMsec": "13707327",
  "isLive": true
}
