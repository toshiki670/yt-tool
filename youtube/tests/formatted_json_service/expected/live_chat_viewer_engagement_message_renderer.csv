id,authorExternalChannelId,postedAt,category,authorName,content
ChwKGkNJQ2RucnJCaUlzREZkM0N3Z1FkcUJRVTJR,,,Viewer engagement message,System Message,message: '新しいゲームはどうですか？<br>持ってる (41%)<br>持ってない (23%)<br>買うか検討中 (17%)<br>絶対に買わない (17%)<br><br>Poll complete: 34 votes'
